src/api/spec/services/workflows/yaml_to_workflows_service_spec.rb

```diff
@@ -83,5 +83,14 @@
         expect { subject }.to raise_error(Token::Errors::WorkflowsYamlNotParsable)
       end
     end
+
+    context 'with invalid placeholder variables' do
+      let(:workflows_yml_file) { Rails.root.join('spec/support/files/unparsable_workflows_placeholders.yml').expand_path }
+      let(:payload) { github_extractor_payload }
+
+      it 'raises a user-friendly error' do
+        expect { subject }.to raise_error(Token::Errors::WorkflowsYamlNotParsable)
```

Comment:
I don't think this is user friendly if it's simply the exception message, which ends up being something like `ArgumentError (malformed format string - %;):`.

Reply:
Oh right, we're re-raising a different exception. I should have paid closer attention :+1: